chia/protocols/solver_protocol.py

```diff
@@ -0,0 +1,21 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+from chia_rs.sized_ints import uint64
+
+from chia.util.streamable import Streamable, streamable
+
+
+@streamable
+@dataclass(frozen=True)
+class SolverInfo(Streamable):
+    plot_strength: uint64
```

Comment:
plot strength is a `uint8`

```suggestion
    plot_strength: uint8
```